ContentProtection/OfflineFairPlay/Program.cs

```diff
@@ -345,7 +345,7 @@ private static async Task RunAsync(ConfigWrapper config)
         /// <returns></returns>
         private static async Task<Asset> CreateOutputAssetAsync(IAzureMediaServicesClient client, string resourceGroupName, string accountName, string assetName)
         {
-            Asset outputAsset = new Asset();
+            Asset outputAsset = new();
```

Comment:
I'm not keen on this style. I think we should be consistent and use `var` everywhere.

Reply:
I liked it :-)

Reply:
Fixed